modules/eco-node-health-check-operator-installation-cli.adoc

```diff
@@ -61,14 +58,17 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
     name: node-health-check-operator
-    namespace: openshift-operators
+    namespace: node-health-check
 spec:
-    channel: alpha
+    channel: candidate <1>
     name: node-healthcheck-operator
     source: redhat-operators
     sourceNamespace: openshift-marketplace
     package: node-health-check-operator
```

Comment:
package: node-healthcheck-operator

Reply:
![image](https://user-images.githubusercontent.com/58551443/155094596-f88d1f35-391d-4592-8d70-b9109a941b25.png)
version in 4.10 will be - 0.2.0

Reply:
![image](https://user-images.githubusercontent.com/58551443/155095285-7ea89836-edb3-45d8-8bbb-ebc9b579cf7a.png)
https://docs.openshift.com/container-platform/4.9/operators/user/olm-installing-operators-in-namespace.html#olm-installing-from-operatorhub-using-web-console_olm-installing-operators-in-namespace
Also, you can add to the subscription: installPlanApproval: Manual 
with comments from the screenshot about installPlanApproval

Reply:
I thought about adding PP to the Example output, since NHC comes with the poison pill operator at installation time together. what do you think?

Reply:
> I thought about adding PP to the Example output, since NHC comes with the poison pill operator at installation time together. what do you think?

Hi Polina, since Poison Pill 4.9 is still not published and 4.10 might be as risk, I am not sure if it's best to include it here in the example output. It might be misleading and may cause confusion. 
Furthermore, this is just an example output for the user to know what to expect.